tests/test_odata.py

```diff
@@ -150,33 +151,11 @@ def test_is_online():
 
 
 @pytest.mark.fast
-def test_parse_odata_missing_content_geometry():
+def test_parse_odata_missing_content_geometry(fixture_path):
     # response from:
     # https://s5phub.copernicus.eu/dhus/odata/v1/Products('e2cc856f-f9e0-4f20-bf04-dd4d890b43c0')?$format=json
+    with open(fixture_path("odata_response_missing_content_geometry.json")) as file:
```

Comment:
don't call this variable `file`, since it masks the built-in. Please use `f` or so instead.

Reply:
You're right, good catch
Outdated though also, since the test no longer exists :smile: